build.gradle

```diff
@@ -29,6 +29,7 @@ buildscript {
         fragmentTestVersion = "1.1.0"
         uiAutomatorTestVersion = "2.2.0"
         annotationTestVersion = "1.2.0"
+        robolectricVersion = "4.4"
```

Comment:
I would try not to add this version here. At the moment we use robolectric just in the library, so there is no need to add this version in the app Gradle from my point of view.